docs/source/release-notes.html.md.erb

```diff
@@ -5,6 +5,10 @@ weight: 2
 
 # Release Notes
 
+## 12 April 2023
+
+- Rename `address3` field to `town` and add new `address3` field. This is a string, referencing an optional address field.
```

Comment:
```suggestion
- We have introduced a new `Street address 3` attribute to the Location schema. This is a string, referencing an optional address field.
```

I don't think we changed the current `city` attribute in the swagger spec so we just need to mention that we've introduced a new public Address 3 field if that makes senses.